Bottom: 8aa0b46a53b12878f7bb4b3e1bcf9eee6f38e208
Top:    eec4fb7a6d90e6dd80544820d3c1930a628d8201
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-07-12 11:12:25 -0500

Move mariadb config to maria role


---
diff --git a/roles/docker-common/tasks/main.yml b/roles/docker-common/tasks/main.yml
index a8ab0c3c..e2a864ab 100644
--- a/roles/docker-common/tasks/main.yml
+++ b/roles/docker-common/tasks/main.yml
@@ -9,7 +9,6 @@
 
 - import_tasks: packages.yml
 - import_tasks: sshd_settings.yml
-- import_tasks: mariadb_settings.yml
 
 - import_tasks: configure.yml
 
diff --git a/roles/docker-common/tasks/mariadb_settings.yml b/roles/docker-common/tasks/mariadb_settings.yml
deleted file mode 100644
index 54508dd3..00000000
--- a/roles/docker-common/tasks/mariadb_settings.yml
+++ /dev/null
@@ -1,15 +0,0 @@
----
-# file: roles/docker-common/tasks/mariadb_settings.yml
-
-- name: Set host ssh ListenAddress
-  template:
-    src: mariadb-bind-address.j2
-    dest: /etc/my.cnf.d/mariadb-bind-address.cnf
-  when: ansible_br0 is defined and ansible_br0.ipv4 is defined and ansible_br0.ipv4.address is defined
-
-- name: Restart MariaDB service
-  service:
-    name: mariadb
-    enabled: yes
-    state: restarted
-  when: ansible_br0 is defined and ansible_br0.ipv4 is defined and ansible_br0.ipv4.address is defined
diff --git a/roles/mariadb/tasks/main.yml b/roles/mariadb/tasks/main.yml
index b27a0af7..70e29ec6 100644
--- a/roles/mariadb/tasks/main.yml
+++ b/roles/mariadb/tasks/main.yml
@@ -42,3 +42,15 @@
     permanent: true
     state: enabled
   when: mariadb_open_firewall
+- name: Set host ssh ListenAddress
+  template:
+    src: mariadb-bind-address.j2
+    dest: /etc/my.cnf.d/mariadb-bind-address.cnf
+  when: ansible_br0 is defined and ansible_br0.ipv4 is defined and ansible_br0.ipv4.address is defined
+
+- name: Restart MariaDB service
+  service:
+    name: mariadb
+    enabled: yes
+    state: restarted
+  when: ansible_br0 is defined and ansible_br0.ipv4 is defined and ansible_br0.ipv4.address is defined
diff --git a/roles/docker-common/templates/mariadb-bind-address.j2 b/roles/mariadb/templates/mariadb-bind-address.j2
similarity index 100%
rename from roles/docker-common/templates/mariadb-bind-address.j2
rename to roles/mariadb/templates/mariadb-bind-address.j2
